plugins/woocommerce/client/admin/client/settings-payments/components/payment-gateway-list/payment-gateway-list.tsx

```diff
@@ -0,0 +1,121 @@
+/**
+ * External dependencies
+ */
+import { PaymentProvider } from '@woocommerce/data';
+import { getAdminLink } from '@woocommerce/settings';
+import { Gridicon } from '@automattic/components';
+
+/**
+ * Internal dependencies
+ */
+import {
+	DefaultDragHandle,
+	SortableContainer,
+	SortableItem,
+} from '~/settings-payments/components/sortable';
+import { PaymentExtensionSuggestionListItem } from '~/settings-payments/components/payment-extension-suggestion-list-item';
+import { PaymentGatewayListItem } from '~/settings-payments/components/payment-gateway-list-item';
+import './payment-gateway-list.scss';
+
+interface PaymentGatewayListProps {
+	providers: PaymentProvider[];
+	installedPluginSlugs: string[];
+	installingPlugin: string | null;
+	setupPlugin: ( id: string, slug: string ) => void;
+	updateOrdering: ( providers: PaymentProvider[] ) => void;
+}
+
+export const PaymentGatewayList = ( {
+	providers,
+	installedPluginSlugs,
+	installingPlugin,
+	setupPlugin,
+	updateOrdering,
+}: PaymentGatewayListProps ) => {
+	return (
+		<SortableContainer< PaymentProvider >
+			items={ providers }
+			className={ 'settings-payment-gateways__list' }
+			setItems={ updateOrdering }
+		>
+			{ providers.map( ( provider: PaymentProvider ) => {
+				switch ( provider._type ) {
+					case 'suggestion':
```

Comment:
Could we use some centralized constants for these so we would have a single, clear place to update in case the BE changes something?

Reply:
Updated in https://github.com/woocommerce/woocommerce/pull/53301/commits/7285d2cdc6aa32ac5607e602ac0ef6033c699594.